### ISSUE TITLE
Ember inspector crashes looking for `ember-web-api/models/web-api`

### ISSUE THREAD

**Issue by cherta** [CrshOverride/ember-web-api#3]
I'm trying to use this adapter but one of the first things I've noticed is that it breaks the ember inspector maybe is my setup or maybe is something related with the inspector itself.

My setup is:

```
-------------------------------
Ember                  : 1.13.7
Ember Data             : 1.13.8
jQuery                 : 1.11.3
Ember Form Master 2000 : 2.1.1
-------------------------------
```

The error I'm getting is the following:

```
Ember Inspector has errored.
This is likely a bug in the inspector itself.
You can report bugs at https://github.com/emberjs/ember-inspector.
Error message: Could not find module `ember-web-api/models/web-api` imported from `patient-portal-web-api/models/web-api`
Stack trace: Error: Could not find module `ember-web-api/models/web-api` imported from `patient-portal-web-api/models/web-api`
    at requireFrom (http://localhost:4200/assets/vendor.js:119:13)
    at reify (http://localhost:4200/assets/vendor.js:106:22)
    at mod.state (http://localhost:4200/assets/vendor.js:149:17)
    at tryFinally (http://localhost:4200/assets/vendor.js:30:14)
    at requireModule (http://localhost:4200/assets/vendor.js:148:5)
    at Ember.DefaultResolver.extend._extractDefaultExport (http://localhost:4200/assets/vendor.js:69471:20)
    at resolveOther (http://localhost:4200/assets/vendor.js:69203:32)
    at superWrapper (http://localhost:4200/assets/vendor.js:32933:20)
    at exports.default._emberRuntimeSystemObject.default.extend.resolve (http://localhost:4200/assets/vendor.js:15520:35)
    at Object.resolve [as resolver] (http://localhost:4200/assets/vendor.js:15276:23)
```

I would really like to help this project because I'm deeply involved in a large app with a .net backend so if you somebody can point me in the right direction I may help myself.

Thanks in advance :)

**Comment by CrshOverride** [comment]
It looks like you've created a `web-api` model in your application and tried to inherit from a `web-api` model from this project (which doesn't actually exist). 

The only thing that this project supplies are the serializer and adapter. You can either use them at the application level (by using the instructions posted in the README.md) or on a model-by-model basis. 

I'm not sure what your local `web-api` model does, but you can probably safely delete it. If you can't delete it, you can remove the inheritance from the nonexistent `web-api` model from this project.

**Comment by cherta** [comment]
Hello.

I have no `web-api` model but now that you mention this my app has `web-api` on the name, that's mainly because I created a new ember app to test a backend we already have. That may be the issue.

I'll change the project name and let you know.

Thanks for the help

**Comment by mmusket** [comment]
I'm running in the same problem. 
Didn't create any models called web-api 

`Deprecations were detected, see the Ember Inspector deprecations tab for more details.
VM910:100 Ember Inspector has errored.
This is likely a bug in the inspector itself.
You can report bugs at https://github.com/emberjs/ember-inspector.
Error message: Could not find module`ember-web-api/models/web-api`imported from`on-my-cv-web/models/web-api`
Stack trace: Error: Could not find module`ember-web-api/models/web-api`imported from`on-my-cv-web/models/web-api`
    at requireFrom (http://localhost:4200/assets/vendor.js:119:13)
    at reify (http://localhost:4200/assets/vendor.js:106:22)
    at mod.state (http://localhost:4200/assets/vendor.js:149:17)
    at tryFinally (http://localhost:4200/assets/vendor.js:30:14)
    at requireModule (http://localhost:4200/assets/vendor.js:148:5)
    at Ember.DefaultResolver.extend._extractDefaultExport (http://localhost:4200/assets/vendor.js:64684:20)
    at resolveOther (http://localhost:4200/assets/vendor.js:64416:32)
    at superWrapper (http://localhost:4200/assets/vendor.js:32125:22)
    at exports.default._emberRuntimeSystemObject.default.extend.resolve (http://localhost:4200/assets/vendor.js:15404:35)
    at Object.resolve [as resolver] (http://localhost:4200/assets/vendor.js:15138:23)`

**Comment by CrshOverride** [comment]
Is this a blank application or something that you've been working on for a while?

I'm trying to figure out the easiest, most reliable reproduction steps so I can start looking into this over the weekend.

**Comment by cherta** [comment]
It happened to me with a blank app, if that helps.

I still need to test removing the `web-api` from the project name, that maybe
helps figure this out.

**Comment by barryofguilder** [comment]
I also ran into this issue. I don't have a model called "web-api" neither do I have anything in my project with that name. I'm using Ember CLI.

**Comment by CrshOverride** [comment]
@mmusket @cherta Which versions of Ember, Ember Data, and Ember CLI are you both using?

**Comment by barryofguilder** [comment]
@CrshOverride 
Ember: 1.13.7
Ember Data: 1.13.8
Ember CLI: 1.13.8

**Comment by cherta** [comment]
Same here @CrshOverride
Ember: 1.13.7
Ember Data: 1.13.8
Ember CLI: 1.13.8

**Comment by CrshOverride** [comment]
Awesome. Thanks all! I'll work on it this weekend and keep everybody updated.

There's also going to be a release shortly that updates everything for compatibility with the new `Serializer`/`Adapter` APIs so keep an eye out for that as well. I'm still working on testing everything.

**Comment by barryofguilder** [comment]
@CrshOverride, awesome! Thanks so much for this add-on, it's made my life so much easier!

**Comment by barryofguilder** [comment]
@CrshOverride for what it's worth, I just updated my app to use the latest Ember (1.13.10) and Ember Data (1.13.12) and the issue still happens.

**Comment by CrshOverride** [comment]
Ok everybody, I've pushed `v0.0.6` to npm. This version upgrades to the new Serializer API for Ember Data 1.13.8. Unfortunately Unit Tests are light (I'm currently working two jobs, cut me some slack, haha) but I've tested in one of my projects and verified that everything was working as expected.

If any of you continue having issues, feel free to let me know. For the time being, I'm closing this issue.

Thanks again for all the feedback!
